MPC/config/dcps_ice.mpb

```diff
@@ -0,0 +1,4 @@
+project: dcps, openssl {
```

Comment:
This is going to have to be `ace_openssl` because of [ACE_TAO#858](https://github.com/DOCGroup/ACE_TAO/pull/858) if this is going to work on Android properly.

Reply:
We will pull out the openssl dependency and put it in the participant.